runtimes/huggingface/mlserver_huggingface/__init__.py

```diff
@@ -1,3 +1,11 @@
 from .runtime import HuggingFaceRuntime
+from .pipelines import StEmbeddingPipeline
+from transformers.pipelines import PIPELINE_REGISTRY
 
+# Added Custom pipeline
+PIPELINE_REGISTRY.register_pipeline(
+    "sentence_embedding",
```

Comment:
nit: could we make it `sentence-embedding`, as that is the format other pipelines use?

Reply:
resolved in the latest commit